.gitignore

```diff
@@ -58,3 +58,5 @@ sysinfo.txt
 # Crashlytics generated file
 crashlytics-build.properties
 
+# MacOS
+.DS_Store
```

Comment:
Is this an auto-gen file specific to MacOS?

Reply:
Yeah Macs generate this file to store OS-specific info about the containing folder like icon position (if you use the free-floating icon mode).

Reply:
For the old DeckBuildingDragon project folder, I think that's left over on your local machine from Unity wanting to re-cache stuff. It doesn't appear in the remote repo as far as I can tell.

I would delete that folder on your machine, and make sure to also remove it from your project list on Unity Hub + add the new root.

Reply:
Ah ok that was it! Thanks, yeah this LGTM.